options/locale/locale_en-US.ini

```diff
@@ -1009,6 +1007,7 @@ broken_message = The Git data underlying this repository cannot be read. Contact
 code = Code
 code.desc = Access source code, files, commits and branches.
 branch = Branch
+branch_plural = Branch{{if not .One}}es{{end}}
```

Comment:
It seems that `{{if .One}}Branch{{else}}Branches{{end}}` could be more friendly to translators.

Reply:
```suggestion
branch_plural = {{if .One}}Branch{{else}}Branches{{end}}
```

Reply:
I meant there are many `word{{if not .One}}s{{end}}` in code.

Do we prefer to keep using `word{{if not .One}}s{{end}}` or using `{{if .One}}word{{else}}words{{end}}`? Personally I would prefer to make it easier for translators.

Either format is preferred, IMO it's better to document it.

Reply:
done.